internal/testbed/scripts/install_dt_collector_helm.sh

```diff
@@ -2,17 +2,25 @@
 echo "Installing Dynatrace Collector using helm"
 helm version
 
-kubectl create namespace otel-collector
-kubectl -n otel-collector create secret generic dynatrace-otelcol-dt-api-credentials --from-literal=DT_API_ENDPOINT=$DT_API_ENDPOINT/api/v2/otlp --from-literal=DT_API_TOKEN=$DT_API_TOKEN
+kubectl create secret generic dynatrace-otelcol-dt-api-credentials --from-literal=DT_ENDPOINT="endpoint" --from-literal=DT_API_TOKEN="token"
 
 helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
 helm repo update
 
-cd .github/actions/deploy-collector
-helm upgrade -i --wait dynatrace open-telemetry/opentelemetry-collector \
-    -f collector-helm-values.yaml \
-    -n otel-collector \
+export TAG=$TAG
+export REPOSITORY=$REPOSITORY
```

Comment:
i'm not sure if this is necessary for envsubst here honestly...

Reply:
works without it, removed